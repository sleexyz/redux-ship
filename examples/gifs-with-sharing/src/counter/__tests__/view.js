// @flow
import React from 'react';
import * as Test from '../../test';
import * as CounterModel from '../model';
import Counter from '../view';

const defaultProps = {
  state: CounterModel.initialState,
};

Test.snapshotComponent(Counter, {
  'default': defaultProps,
});
