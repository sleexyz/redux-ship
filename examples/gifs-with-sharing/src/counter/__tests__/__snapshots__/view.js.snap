exports[`test default 1`] = `
<div>
  <p>
    Count: 
    0
  </p>
</div>
`;
